apps/web/utils/server-action.ts

```
@@ -1,7 +1,12 @@
 "use client";
 
 import { toastError, toastSuccess } from "@/components/Toast";
-import { type ServerActionResponse, isActionError } from "@/utils/error";
+import {
+  ActionError,
+  type ServerActionResponse,
+  captureException,
+  isActionError,
+} from "@/utils/error";
```

Biome

[error] Some named imports are only used as types.

This import is only used as a type.

Importing the types with import type ensures that they are removed by the transpilers and avoids loading unnecessary modules.
Safe fix: Use import type.


(lint/style/useImportType)